inc/endpoints/class-cache-endpoint.php

```diff
@@ -62,9 +66,15 @@ public function get_route_response( $request ) {
 				// to firing the action.
 				if ( ! empty( $route ) ) {
 					do_action( 'wp_irving_page_cache_purge', $route );
+
+					return $response;
 				}
 			}
 		}
+
+		$response = new \WP_REST_Response( false );
```

Comment:
It might be worth returning a `WP_Error` object in this case with an error code and the error string from the JS, rather than a `WP_REST_Response`. The error will automatically be converted to a REST response, and WordPress will automatically set the HTTP response code is `500`, which might be more appropriate (a different HTTP response code can also be passed to the `WP_Error`). The downside is that `formatAlert()` will be a little more complicated because it needs to check for the error response.